server/lib/parties/party-service.test.ts

```diff
@@ -451,4 +476,39 @@ describe('parties/party-service', () => {
       expect(client2.unsubscribe).toHaveBeenCalledWith(getPartyPath(party.id))
     })
   })
+
+  describe('sendChatMessage', () => {
+    let leader: PartyUser
+    let party: PartyRecord
+
+    beforeEach(async () => {
+      leader = user1
+      party = await partyService.invite(leader, USER1_CLIENT_ID, user2)
+      partyService.acceptInvite(party.id, user2, USER2_CLIENT_ID)
+    })
+
+    test('should throw if the party is not found', () => {
+      expect(() =>
+        partyService.sendChatMessage('INVALID_PARTY_ID', user2.id, 'Hello World!'),
+      ).toThrowErrorMatchingInlineSnapshot(`"Party not found"`)
+    })
+
+    test('should throw if the user is not in party', () => {
+      expect(() =>
+        partyService.sendChatMessage(party.id, user3.id, 'Hello World!'),
+      ).toThrowErrorMatchingInlineSnapshot(`"User not in party"`)
+    })
+
+    test('should publish "chatMessage" event to the party path', () => {
+      Date.now = jest.fn(() => 272727)
```

Comment:
Does this get reset between tests? My guess is probably not, so you should probably do this in a `before` and clean it up in an `after` (or, better yet, don't do this at all and fix the code to allow this to be passed in or at the very least inject the thing that retrieves the current date)